### PR TITLE
Introduce SSL config for Kafka

### PR DESCRIPTION
**Pull request by gadomsky**

### CODE REVIEW

**Comment by piotrrzysko** [comment]
Hi @gadomsky, would you mind rebasing this branch on the latest master?

**Comment by gadomsky** [comment]
Hi @piotrrzysko, can you please add required labels to PR, so build can pass? Also, I have rebased it, so it's ready for review.

File: hermes-management/src/main/java/pl/allegro/tech/hermes/management/config/kafka/KafkaSSLProperties.java [hunk 1]
```diff
@@ -0,0 +1,197 @@
+package pl.allegro.tech.hermes.management.config.kafka;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "ssl")
+public class KafkaSSLProperties {
```
Comment (faderskd) [inline hermes-management/src/main/java/pl/allegro/tech/hermes/management/config/kafka/KafkaSSLProperties.java:6]: in fact it is a copy of the `pl.allegro.tech.hermes.common.config.KafkaSSLProperties` properties. Why we can't use that ?

File: hermes-common/src/main/java/pl/allegro/tech/hermes/common/kafka/KafkaConsumerPoolConfig.java [hunk 2]
```diff
@@ -12,20 +12,122 @@ public class KafkaConsumerPoolConfig {
...
+    private final boolean isSslEnabled;
+    private final String sslTrustStoreLocation;
+    private final String sslTrustStorePassword;
+    private final String sslKeyStoreLocation;
+    private final String sslKeyStorePassword;
+    private final String sslKeyPassword;
+    private final String sslProtocolVersion;
+    private final String sslEndpointIdentificationAlgorithm;
+
+    public String getSslKeyStoreCertificateChain() {
```
Comment (piotrrzysko) [inline hermes-common/src/main/java/pl/allegro/tech/hermes/common/kafka/KafkaConsumerPoolConfig.java:24]: Let's keep the order of new fields and getters/setters consistent with the existing ones.

**Pull request closed without merging**